README.md

```diff
@@ -9,26 +9,31 @@ For more details about the design and architecture of the SDK, see the [official
 ### Required Prerequisites
 To use this SDK you must have:
 
-* **A Java 8 development environment**
+* **A Java 8 or newer development environment**
 
-  If you do not have one, go to [Java SE Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html) on the Oracle website, then download and install the Java SE Development Kit (JDK). Java 8 or higher is required.
+  If you do not have one, we recommend [Amazon Corretto](https://aws.amazon.com/corretto/).
 
   **Note:** If you use the Oracle JDK, you must also download and install the [Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html).
 
-* **Bouncy Castle**
+* **Bouncy Castle** or **Bouncy Castle FIPS**
 
-  Bouncy Castle provides a cryptography API for Java. If you do not have Bouncy Castle, go to https://bouncycastle.org/latest_releases.html, then download the provider file that corresponds to your JDK. Or, you can pick it up from Maven:
+  The AWS Encryption SDK for Java uses Bouncy Castle for serializing and deserializing cryptographic objects.
+  It does not explicitly use Bouncy Castle (or any other [JCA Provider](https://docs.oracle.com/javase/8/docs/api/java/security/Provider.html)) for the underlying cryptography.
+  Instead, it uses the platform default, which may be configured or overridden as documented in the
```

Comment:
The developer using the library is the actor here, right?

"Instead, it uses the platform default, which you can configure or override as documented in the..."

Reply:
Yup. Can change this.